app/src/main/java/com/konkuk/medicarecall/ui/feature/statistics/viewmodel/StatisticsViewModel.kt

```diff
@@ -216,6 +218,13 @@ class StatisticsViewModel @Inject constructor(
         val sleepH = averageSleep.hours
         val sleepM = averageSleep.minutes
 
+        // UI 색상 문제 대응 TODO: 서버에서 null로 변경시 해당 로직 제거
+        val noteText = if (healthSummary == "아직 충분한 기록이 쌓이지 않았어요.") {
+            ""
+        } else {
+            healthSummary ?: ""
+        }
+
```

Comment:
```suggestion
        val noteText = if (healthSummary == "아직 충분한 기록이 쌓이지 않았어요." || healthSummary == null) ""
        else healthSummary
```
두 조건문을 하나의 조건문으로 합칠 수 있을 것 같습니다!

다른 방법으로는 아래처럼 할 수도 있어요
```kotlin
        val noteText = healthSummary.takeIf { it != "아직 충분한 기록이 쌓이지 않았어요." && it != null } ?: ""
```